### PR TITLE
For BYOC Azure accounts the powershell script is needed to allow automatically deletion of temporal users

### PR DESCRIPTION
**Pull request by julianatienza**
@gsanchezu Please, can you review new Doc. with me before to ask for merge? Thanks in advance!

### CODE REVIEW

**Comment by jim-phillips** [comment]
@gsanchezu is this good to go?

File: Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md [hunk 1]
```diff
@@ -48,17 +48,30 @@ You need an Microsoft Azure subscription to be able to consume Azure services. F
...
-
-    Then click **Save**.
-    Now click on **Add > Windows Azure Active Directory** and select the following **Application** permission:
-    * Read and write directory data _(Directory.ReadWrite.All)_
+    * Access directory as the signed in user
+    _(Directory.AccessAsUser.All)_
 
     Then click **Save** and **Grant permissions** to apply them to your application.
 
+    In order to allow automatically deletion, you must add the Company Administrator Role to the App. https://docs.microsoft.com/en-us/powershell/module/msonline/add-msolrolemember?view=azureadps-1.0 :
```
Comment (gsanchezu) [inline Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md]: Please, change **automatically** by **automatic**

Comment (julianatienza) [reply Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md]: Ok! Thx!

**Comment by julianatienza** [comment]
@jim-phillips  now it is good. You can merge. Thx

**Pull request merged**